paparazzi/paparazzi/build.gradle

```diff
@@ -26,8 +26,12 @@ dependencies {
 dependencies {
   // Paparazzi is a Kotlin JVM module and thus cannot depend on Android library artifacts (AARs)
   // TODO: Use Gradle transforms to extract/rename classes.jar's from AARs
-  api files('libs/compose-runtime-1.3.1.jar')
-  compileOnly files('libs/compose-ui-1.3.1.jar')
+  api files('libs/compose-runtime-1.3.3.jar')
+  api files('libs/compose-ui-1.3.3.jar')
+  compileOnly files('libs/ui-geometry-1.3.3.jar')
+  compileOnly files('libs/ui-graphics-1.3.3.jar')
+  compileOnly files('libs/ui-text-1.3.3.jar')
+  compileOnly files('libs/ui-unit-1.3.3.jar')
```

Comment:
is there anything specific to these libs needed in this PR for compose accessibility?  if not, i'd prefer to move this out of this PR to keep the scope focused.

Reply:
yes I was able to remove `graphics` and `unit` libs. `geometry` is needed to get composable bounds and `text` for use of `AnnotatedString`